docs/how-to-run.adoc

```diff
@@ -51,126 +156,92 @@ $ ./tools/install_runner_prerequisites.sh
-"config-rss.toml" file that you're using.  This must happen before packaging up
-the sled agent (which is the next step below).
-
-If you don't specify initial certificates in this way, you can always load
-certificates later via the API.  This assumes you have a way to reach the API
-that doesn't require a valid TLS certificate.  Today, Nexus always starts an
-HTTP server that you can use for this.  This may be removed in the future.
-
-=== Configuration
-
-Update your `smf/seld-agent/$MACHINE/config-rss.toml` with the appropriate network information:
+The relevant configuration files are in `./smf/sled-agent/$MACHINE`.  Start with `config-rss.toml` in one of those directories.  There are only a few parts you need to review:
 
+[source,toml]
 ----
 [[internal_services_ip_pool_ranges]]
-first = "192.168.1.20"          <--- first address of pool used for internal services (like Nexus)
-last = "192.168.1.22"           <--- last address of pool used for internal services
+first = "192.168.1.20"
+last = "192.168.1.29"
+----
 
-# Configuration to bring up boundary services and make nexus reachable from the outside
+This is a range of IP addresses on your external network that Omicron can assign to externally-facing services (like DNS and the API).  You'll need to change these if you've picked different addresses for your external network.  See <<_external_networking>> above for more on this.
+
+[source,toml]
+----
+# Configuration to bring up boundary services and make Nexus reachable from the
+# outside.  This block assumes that you're following option (2) above: putting
+# your Oxide system on an existing network that you control.
 [rack_network_config]
-gateway_ip = "192.168.1.199"    <--- ip address of your default gateway / router
-infra_ip_first = "192.168.1.30" <--- first address of pool used for rack networking
-infra_ip_last = "192.168.1.50"  <--- last address of pool used for rack networking
-uplink_port = "qsfp0"           <--- for "softnpu" environments, this will always be "qsfp0"
-uplink_port_speed = "40G"       <--- uplink interface speed
-uplink_ip = "192.168.1.32"      <--- address to assign to the uplink port
+# The gateway for the external network
+gateway_ip = "192.168.1.199"
+# A range of IP addresses used by Boundary Services on the network.  In a real
+# system, these would be addresses of the uplink ports on the Sidecar.  With
+# softnpu, only one address is used.
+infra_ip_first = "192.168.1.30"
+infra_ip_last = "192.168.1.30"
+# Name of the port.  This should always be "qsfp0" when using softnpu.
+uplink_port = "qsfp0"
+uplink_port_speed = "40G"
+uplink_port_fec="none"
+# For softnpu, an address within the "infra" block above that will be used for
+# the softnpu uplink port.  You can just pick the first address in that pool.
+uplink_ip = "192.168.1.30"
 ----
 
 Update your `smf/sled-agent/$MACHINE/config.toml` if necessary:
+
+TODO how does one know if this is necessary and if so, what to put here?
```

Comment:
@rcgoodfellow could you help flesh this out?  If it's easier, I'm happy to get a brain dump from you and then write up a summary here.

Reply:
I don't think any modifications to this file are necessary for basic Omicron development environments.

The `switch_zone_maghemite_links` field is necessary if you are running a Canada-region style setup where you plumb physical interfaces into the switch zone, and `ddmd` in that zone needs to actually peer with other physical machines on your lab network. @jmpesp will likely have more insight here.